core/services/nmea_injector/main.py

```diff
@@ -0,0 +1,104 @@
+#! /usr/bin/env python3
+import argparse
+import asyncio
+import logging
+from typing import Any, List
+
+from commonwealth.utils.apis import PrettyJSONResponse
+from commonwealth.utils.logs import InterceptHandler
+from fastapi import FastAPI, Response, status
+from fastapi.responses import HTMLResponse
+from fastapi_versioning import VersionedFastAPI, version
+from loguru import logger
+from uvicorn import Config, Server
+
+from TrafficController import NMEASocket, SocketKind, TrafficController
+
+parser = argparse.ArgumentParser(description="I'm here! service for Blue Robotics Companion")
```

Comment:
The name should be changed to "NMEA Injector", also check the others parts to be sure that we cover all the cases of the old name.